src/context/wishListContext.js

```diff
@@ -0,0 +1,83 @@
+import { createContext, useContext, useReducer } from "react";
+import { useFetchApi } from "../hooks/useFetchApi";
+import { useAuth } from "./";
+import { initialSharedState } from "../reducers/constants";
+import { sharedReducer } from "../reducers/reducerFunctions/sharedReducer";
+import jwt from "jsonwebtoken";
```

Comment:
I think you have not used this `jwt` anywhere so you can remove this import statement.

Reply:
Yes I forgot to remove this will remove it now.